getDataTableReader.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Get Data (TableDataReader)</title>
+
+    <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
```

Comment:
Consider removing the dependency on jQuery since you're really only using it to get the value of dropdown. Without it, you could get the value with raw JS instead by:

```html
<option value="0" selected disabled>Select page</option>
```

```js
const pageSelector = document.getElementById('pageSelect');
const pageNumber = parseInt(pageSelector.value, 10);
```

Reply:
Yes. That's much better.  I was able to remove the dependency on jQuery. Thanks.